src/Employer/Employer.Web/Views/Title/Title.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@using Esfa.Recruit.Employer.Web.ViewModels.Part1.TitleVacancy
+@model Esfa.Recruit.Employer.Web.ViewModels.Part1.TitleVacancy.TitleViewModel
+
+<div class="grid-row">
+    <div class="column-full">
+        
+        @Html.Partial("_Part1Header")
+
+        <!-- Validation summary-->
+        <div asp-validation-summary="All" class="error-summary" role="alert" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
+            <h2 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
+                Please fix these errors
+            </h2>
+        </div>
+
+        <!-- Title -->
+        <div id="section_title">
+            <form asp-route="@(Model.VacancyId.HasValue ? RouteNames.Title_Post : RouteNames.CreateVacancy_Post)">
+                <div class="form-group">
+
+                    <label asp-for="Title" class="form-label-bold">
+                        What do you want to call this vacancy?
+                        <span class="form-hint">For example, Software developer apprenticeship</span>
+                    </label>
+                    <input asp-for="Title" class="form-control-3-4 form-control" size="12" type="text" maxlength="50" data-val-length-max="100"/>

```

Comment:
maxlength 50?

Reply:
The story was updated after my PR. I'll revisit now